### PR TITLE
Docs: Fix author email description

### PR DESCRIPTION
**Pull request by dbbaughe**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Good catch! 👍 

Since the `metadata.md` file is a generated file, you'd have to make the change in [plugin.schema.json](https://github.com/grafana/grafana/blob/10a942aad0498e9e9d4530d008751941bfec6511/docs/sources/developers/plugins/plugin.schema.json#L190) and then generate the metadata.md using [json-schema-docs](https://github.com/grafana/json-schema-docs).

Could you update plugin.schema.json as well?

**Comment by dbbaughe** [comment]
Hey @marcusolsson,

Let me know if I used your json schema docs incorrectly...
`../json-schema-docs/json-schema-docs -schema ./docs/sources/developers/plugins/plugin.schema.json > ./docs/sources/developers/plugins/metadata.md`

It seems to have removed the notice at the top of the metadata.md docs to not edit the file (which I did not see before 😉)

**Comment by marcusolsson** [comment]
Sorry about that 🙏  we need to improve the documentation for how to update the plugin schema docs.

You need to use the template to add the front matter:

```
json-schema-docs -schema ./plugin.schema.json -template ./metadata.md.tpl > metadata.md
```

File: docs/sources/developers/plugins/metadata.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
-+++
-# -------------------------------------------------------------------------
-# Do not edit this file. It is automatically generated by json-schema-docs.
-# -------------------------------------------------------------------------
-title = "plugin.json"
-keywords = ["grafana", "plugins", "documentation"]
-aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
-+++
-
```
Comment (achatterjee-grafana) [inline docs/sources/developers/plugins/metadata.md:9]: Why was the front matter deleted? Needs dev review.

**Review by achatterjee-grafana — COMMENTED** [review]
Added a question.

**Comment by dbbaughe** [comment]
@marcusolsson, feel free to pick up the change. I think I should have just opened a simple GitHub issue instead :P
There was a bit more involvement than I thought around the CLA, so going to close this on my end for now.

Thanks!

**Pull request closed without merging**